lib/ui/prompt.js

```diff
@@ -72,8 +72,9 @@ PromptUI.prototype.processQuestion = function (question) {
 };
 
 PromptUI.prototype.fetchAnswer = function (question) {
+  var screen = this.createScreen();
   var Prompt = this.prompts[question.type];
-  var prompt = new Prompt(question, this.rl, this.answers);
+  var prompt = new Prompt(question, screen, this.rl, this.answers);
```

Comment:
I wonder if it'd make sense to store the prompt instead of the screen; eg`this.activePrompt = new Prompt();`

Then we could call `this.activePrompt.stop()` and that function would be in charge of clearing the screen. It's an extra layer, but it'd prevent having to pass `this.screen` as argument everywhere.